english/yell-at-snape/yell-at-snape.md

```diff
@@ -1,5 +1,7 @@
-Hermione, her distaste for Snape's rule-breaking 
```

Comment:
I realize that editing the text wholesale and copying it to the file is not the best approach as it obfuscates exactly what I changed. 

Changes
1. Added comma after 'professors'
2. Fixed spelling of 'unforgivable'
3. Added new option